seclea_ai/seclea_ai.py

```diff
@@ -21,13 +21,13 @@
     assemble_dataset,
 )
 from .exceptions import AuthenticationError
+from .lib.seclea_utils.core.transformations import encode_func, decode_func
```

Comment:
Can't comment directly on the submodule so doing it on this line. 

The submodule references the wrong commit - needs to be on SPT-2005 to be correct this ref is dev latest.

Reply:
This has now been resolved